### PR TITLE
Updates and Edits to Contributing.md

### PR DESCRIPTION
**Pull request by JayStag**
@acrosman Hello! I made some edits to the copy and included some external links to the Contributing.md file. I also include the standard GitHub workflow. Let me know what you think. Happy to iterate if you like where this document is going. Cheers! 🙂

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -1,9 +1,64 @@
-I welcome contributions large and small, and I'm happy to help new developers gets started working on public code.
+<h1>Contributing to SC_Salary_Data</h1>
```
Comment (acrosman) [inline Contributing.md]: [Please use Markdown](https://github.github.com/gfm/#example-85) instead of an h1 tag.

File: Contributing.md [hunk 2]
```diff
@@ -1,9 +1,64 @@
...
+<hr>
```
Comment (acrosman) [inline Contributing.md]: [Please use markdown](https://github.github.com/gfm/#example-85) instead of hr tag.

**Review by acrosman — CHANGES_REQUESTED** [review]
Thank you again for offering this improvement.  Please you make sure that anything that could be markdown is in Markdown instead of HTML.

File: Contributing.md (hunk 1 above)
Comment (JayStag) [reply Contributing.md]: Will do. 👍

**Review by acrosman — APPROVED** [review]
Thank you for making those changes.

**Pull request merged**